package.json

```diff
@@ -6,6 +6,9 @@
     "@testing-library/jest-dom": "^5.1.1",
     "@testing-library/react": "^10.0.1",
     "@testing-library/user-event": "^10.0.0",
```

Comment:
The testing stuff should go to dev dependencies to not bloat the production build

Reply:
`"@testing-library/jest-dom": "^5.1.1",` will throw a ESLint error, but the rest were moved.